### ISSUE TITLE
Any reason FileDownload is windows only?

### ISSUE THREAD

**Issue by TylerLeonhardt** [PowerShellOrg/PSDepend#98]
I would think we could just use the web cmdlets [here](https://github.com/RamblingCookieMonster/PSDepend/blob/master/PSDepend/Private/Get-WebFile.ps1)?

I wanted to use FileDownload in a project but sadly can't because it's windows only :(

**Comment by RamblingCookieMonster** [comment]
Nope!  We could definitely swap that out for something xplat, or just add some conditional logic per patform!
